circleci: failed tests: TestShowCreateTable

The following test appears to have failed:

[#16665](https://circleci.com/gh/cockroachdb/cockroach/16665):

```
1      storage/intent_resolver.go:331
I160422 05:07:08.603851 /go/src/google.golang.org/grpc/clientconn.go:462  grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:48461: getsockopt: connection refused"; Reconnecting to "127.0.0.1:48461"
I160422 05:07:10.375073 /go/src/google.golang.org/grpc/clientconn.go:462  grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:48461: getsockopt: connection refused"; Reconnecting to "127.0.0.1:48461"
I160422 05:07:10.375301 /go/src/google.golang.org/grpc/clientconn.go:317  Failed to dial 127.0.0.1:48461: grpc: timed out trying to connect; please retry.
I160422 05:07:10.375372 storage/engine/rocksdb.go:178  closing in-memory rocksdb instance
--- FAIL: TestShowCreateTable (3.11s)
	show_test.go:110: pq: sql/lease.go:108: table ID 52 not found
=== RUN   TestSplitOnTableBoundaries
W160422 05:07:10.434511 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
I160422 05:07:10.435098 storage/engine/rocksdb.go:137  opening in memory rocksdb instance
W160422 05:07:10.436078 gossip/gossip.go:892  not connected to cluster; use --join to specify a connected node
I160422 05:07:10.436264 server/node.go:359  store store=0:0 ([]=) not bootstrapped
I160422 05:07:10.437459 raft/raft.go:469  [group 1] 1 became follower at term 5
I160422 05:07:10.437542 raft/raft.go:238  [group 1] newRaft 1 [peers: [1], term: 5, commit: 10, applied: 10, lastindex: 10, lastterm: 5]
I160422 05:07:10.437566 raft/raft.go:549  [group 1] 1 is starting a new election at term 5
I160422 05:07:10.437593 raft/raft.go:482  [group 1] 1 became candidate at term 6
--
I160422 05:07:13.322213 /go/src/google.golang.org/grpc/clientconn.go:462  grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:39827: getsockopt: connection refused"; Reconnecting to "127.0.0.1:39827"
I160422 05:07:15.108139 /go/src/google.golang.org/grpc/clientconn.go:462  grpc: Conn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp 127.0.0.1:39827: getsockopt: connection refused"; Reconnecting to "127.0.0.1:39827"
I160422 05:07:15.108361 /go/src/google.golang.org/grpc/clientconn.go:317  Failed to dial 127.0.0.1:39827: grpc: timed out trying to connect; please retry.
I160422 05:07:15.108437 storage/engine/rocksdb.go:178  closing in-memory rocksdb instance
--- PASS: TestRollbackInRestartWait (3.07s)
FAIL
FAIL	github.com/cockroachdb/cockroach/sql	54.361s
=== RUN   TestEval
--- PASS: TestEval (0.01s)
=== RUN   TestEvalError
--- PASS: TestEvalError (0.00s)
=== RUN   TestEvalComparisonExprCaching
--- PASS: TestEvalComparisonExprCaching (0.00s)
=== RUN   TestSimilarEscape
--- PASS: TestSimilarEscape (0.00s)
=== RUN   TestClusterTimestampConversion
--- PASS: TestClusterTimestampConversion (0.00s)

```
Please assign, take a look and update the issue accordingly.

--- a/sql/show_test.go
+++ b/sql/show_test.go
@@ -42,13 +42,13 @@ func TestShowCreateTable(t *testing.T) {
 		expect string // empty means identical to stmt
 	}{
 		{
-			stmt: `CREATE TABLE t (
+			stmt: `CREATE TABLE %s (
 	i INT CHECK (i > 0),
 	s STRING NULL,
 	v FLOAT NOT NULL,
 	t TIMESTAMP DEFAULT NOW()
 )`,
-			expect: `CREATE TABLE T (
+			expect: `CREATE TABLE %s (
 	i INT NULL CHECK (i > 0),
 	s STRING NULL,
 	v FLOAT NOT NULL,
@@ -56,42 +56,45 @@ func TestShowCreateTable(t *testing.T) {
 )`,
 		},
 		{
-			stmt: `CREATE TABLE T (
+			stmt: `CREATE TABLE %s (
 	i INT PRIMARY KEY
 )`,
-			expect: `CREATE TABLE T (
+			expect: `CREATE TABLE %s (
 	i INT NOT NULL,
 	CONSTRAINT "primary" PRIMARY KEY (i)
 )`,
 		},
 		{
 			stmt: `
-				CREATE TABLE T (i INT, f FLOAT, s STRING, d DATE);
-				CREATE INDEX idx_if on T (f, i) STORING (s, d);
-				CREATE UNIQUE INDEX on T (d);
+				CREATE TABLE %s (i INT, f FLOAT, s STRING, d DATE);
+				CREATE INDEX idx_if on %[1]s (f, i) STORING (s, d);
+				CREATE UNIQUE INDEX on %[1]s (d);
 			`,
-			expect: `CREATE TABLE T (
+			expect: `CREATE TABLE %s (
 	i INT NULL,
 	f FLOAT NULL,
 	s STRING NULL,
 	d DATE NULL,
 	INDEX idx_if (f, i) STORING (s, d),
-	UNIQUE INDEX T_d_key (d)
+	UNIQUE INDEX %[1]s_d_key (d)
 )`,
 		},
 		{
-			stmt: `CREATE TABLE T (
+			stmt: `CREATE TABLE %s (
 	"te""st" INT NOT NULL,
 	CONSTRAINT "pri""mary" PRIMARY KEY ("te""st")
 )`,
 		},
 	}
-	const name = "T"
-	for _, test := range tests {
+	for i, test := range tests {
+		name := fmt.Sprintf("T%d", i)
+		fmt.Println("NAME", name)
 		if test.expect == "" {
 			test.expect = test.stmt
 		}
-		if _, err := sqlDB.Exec(test.stmt); err != nil {
+		stmt := fmt.Sprintf(test.stmt, name)
+		expect := fmt.Sprintf(test.expect, name)
+		if _, err := sqlDB.Exec(stmt); err != nil {
 			t.Fatal(err)
 		}
 		row := sqlDB.QueryRow(fmt.Sprintf("SHOW CREATE TABLE %s", name))
@@ -102,23 +105,25 @@ func TestShowCreateTable(t *testing.T) {
 		if scanName != name {
 			t.Fatalf("expected table name %s, got %s", name, scanName)
 		}
-		if create != test.expect {
-			t.Fatalf("statement: %s\ngot: %s\nexpected: %s", test.stmt, create, test.expect)
+		if create != expect {
+			t.Fatalf("statement: %s\ngot: %s\nexpected: %s", stmt, create, expect)
 			continue
 		}
 		if _, err := sqlDB.Exec(fmt.Sprintf("DROP TABLE %s", name)); err != nil {
 			t.Fatal(err)
 		}
 		// Re-insert to make sure it's round-trippable.
-		if _, err := sqlDB.Exec(test.expect); err != nil {
-			t.Fatalf("reinsert failure: %s: %s", test.expect, err)
+		name += "_2"
+		expect = fmt.Sprintf(test.expect, name)
+		if _, err := sqlDB.Exec(expect); err != nil {
+			t.Fatalf("reinsert failure: %s: %s", expect, err)
 		}
 		row = sqlDB.QueryRow(fmt.Sprintf("SHOW CREATE TABLE %s", name))
 		if err := row.Scan(&scanName, &create); err != nil {
 			t.Fatal(err)
 		}
-		if create != test.expect {
-			t.Errorf("round trip statement: %s\ngot: %s", test.expect, create)
+		if create != expect {
+			t.Errorf("round trip statement: %s\ngot: %s", expect, create)
 			continue
 		}
 		if _, err := sqlDB.Exec(fmt.Sprintf("DROP TABLE %s", name)); err != nil {

